server/src-lib/Hasura/Eventing/EventTrigger.hs

```diff
@@ -264,7 +265,7 @@ processEventQueue logger logenv httpMgr pool getSchemaCache eeCtx@EventEngineCtx
             -- For such an event, we unlock the event and retry after a minute
             setRetry e (addUTCTime 60 currentTime)
           >>= flip onLeft logQErr
-        Just eti -> do
+        Just eti -> runTraceT ("Event trigger: " <> unNonEmptyText (unTriggerName (etiName eti))) do
```

Comment:
I'm not sure I need the prefix `Event trigger: ` here really. What do people think?

Reply:
I wouldn't mind seeing this if I'm debugging something.